### DISCUSSION TITLE
"Leaflet" on plates?

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#568] [Get Help] [answered]
Hi all! Someone on my team just pointed out that on all of our Plates in the lower right corner of the image half of the side-by-side layout is a small gray box that says "leaflet." See screenshot:

![Screenshot 2022-11-02 094924](https://user-images.githubusercontent.com/74433007/199521881-b0de7dd7-b779-4585-8ecc-70de66feb54d.png)

What is this and how can we make it go away?

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @mmpadilla15 [Leaflet](https://leafletjs.com/) is the technology that powers Quire's image viewer. Their name is essentially showing up as a way for the organization to get some credit for providing the viewer as an open-source tool. 

You should be able to remove the text by adding some CSS to your custom.css file:
 
```
.leaflet-control-attribution.leaflet-control {
  display: none;
}
```

Let us know if this works!

**Reply by mmpadilla15** [reply]
@Erin-Cecele That doesn't seem to be working. Any other thoughts?

**Reply by mmpadilla15** [reply]
@Erin-Cecele Nevermind! I tried a hard refresh and it's working now!

**Reply by Erin-Cecele** [reply]
Wonderful. Thank you for confirming @mmpadilla15. Love that hard refresh ;-)
